docs/0_setting-up.md

```diff
@@ -1,12 +1,12 @@
-# 👶 Setting up
+# 🔨 Setting up
 ## 📝 Requirements
 - Android 8+
-- ARM64 device
+- ARM64 device with ARMv8 *(Binder64 devices will NOT work)*
```

Comment:
This is too technical.

Reply:
Isn't that what documentation are supposed to be? It has to contain all the information if possible.